src/structures/MessageSelectMenu.js

```diff
@@ -94,6 +101,45 @@ class MessageSelectMenu extends BaseMessageComponent {
     return this;
   }
 
+  /**
+   * Sets the type of the select menu
+   * @param {SelectMenuComponentType} type Type of the select menu
+   * @returns {MessageSelectMenu}
+   */
+  setType(type) {
+    if (!MessageSelectMenu.checkType(type)) throw new TypeError('INVALID_TYPE', 'type', 'SelectMenuComponentType');
+    this.type = BaseMessageComponent.resolveType(type);
+    return this;
+  }
+
+  /**
+   * Adds the channel type to the select menu
+   * @param {...ChannelType[]} channelType Added channel type
+   * @returns {MessageSelectMenu}
+   */
+  addChannelTypes(...channelTypes) {
+    // eslint-disable-next-line max-len, prettier/prettier
+    if (!channelTypes.every(channelType => ChannelTypes[channelType])) throw new TypeError('INVALID_TYPE', 'channelTypes', 'Array<ChannelType>');
```

Comment:
make it two line instead of disabling 2 rules

Reply:
then its
![image](https://user-images.githubusercontent.com/76836550/200127941-6138b750-af2f-4a63-a896-4f826132645c.png)
works in `{ }` but with comment its clearer